composer.json

```diff
@@ -3,10 +3,21 @@
         {
             "name": "Tom",
             "email": "tomjharding@live.co.uk"
+        },
+        {
+            "name": "Woody Gilk",
+            "email": "woody.gilk@gmail.com",
+            "role": "Contributor"
         }
     ],
     "autoload": {
-        "files": ["src/Combinators.php"]
+        "psr-4": {
+            "PhpFp\\Combinators\\": "src/"
+        },
+        "files": [
+            "src/functions.php",
+            "src/curried.php"
```

Comment:
They have to be imported somehow, or PHP won't find them. However, both are contained in namespaces so they won't pollute the global namespace.